templates/initial-setup/connection_profile.php

```diff
@@ -61,139 +57,64 @@
-					  <a href="https://auth0.com/docs/cms/wordpress/installation#option-1-standard-setup" target="_blank">
-							<?php _e( 'Detailed instructions for this option are here.', 'wp-auth0' ); ?>
-						</a>
-					</p>
-					<p><a class="a0-button primary" href="#" id="manuallySetToken"><?php _e( 'Start Standard Setup', 'wp-auth0' ); ?></a></p>
-					<br>
-
-					<h4><?php _e( 'User Migration Setup', 'wp-auth0' ); ?></h4>
-					<p>
-						<?php _e( 'This includes everything above plus data migration from your WordPress database.', 'wp-auth0' ); ?>
-						<?php _e( 'This requires an inbound connection from Auth0 servers and cannot be changed later without losing data.', 'wp-auth0' ); ?>
-					  <a href="https://auth0.com/docs/cms/wordpress/how-does-it-work#scenario-data-migration" target="_blank">
-							<?php _e( 'More information here.', 'wp-auth0' ); ?>
-					  </a>
-					</p>
-
-					<p>
-						<a class="a0-button primary submit" href="#" id="automaticSetup">
-							<?php _e( 'Start User Migration Setup', 'wp-auth0' ); ?>
-						</a>
-					</p>
-					<br>
-
-				  <h4><?php _e( 'Manual Setup', 'wp-auth0' ); ?></h4>
-				  <p><?php _e( 'If you already have an Application or want to use an existing database connection, please follow the steps below.', 'wp-auth0' ); ?></p>
-				  <p><a class="a0-button primary" href="https://auth0.com/docs/cms/wordpress/installation#manual-setup"
-						target="_blank"><?php _e( 'manual Setup Instructions', 'wp-auth0' ); ?></a></p>
-				  <br>
-			  </div>
-			</div>
-		  </div>
-		</div>
-
-		<div class="modal fade" id="enterTokenModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
-		  <div class="modal-dialog" role="document">
-			<div class="modal-content">
-			  <div class="modal-header">
-				<button type="button" class="close" data-dismiss="modal" aria-label="<?php _e( 'Close', 'wp-auth0' ); ?>">
-					<span aria-hidden="true">&times;</span>
-				</button>
-				<h4 class="modal-title" id="enterTokenModalModalLabel"><?php _e( 'Management API Token', 'wp-auth0' ); ?></h4>
-			  </div>
-			  <div class="modal-body">
-				<p>
-					<?php _e( 'Enter your tenant Domain:', 'wp-auth0' ); ?>
-				</p>
-				<input type="text" name="domain" class="js-a0-setup-input" placeholder="tenant-name.auth0.com" required>
-				  <p>
-						<a href="https://auth0.com/docs/getting-started/the-basics#domains" target="_blank">
-							<small><?php _e( 'More about tenant Domains here', 'wp-auth0' ); ?></small>
-						</a>
-				  </p>
-				<br><br>
-				<p>
-				  <a href="https://auth0.com/docs/api/management/v2/get-access-tokens-for-test#get-access-tokens-manually"
-						target="_blank">
-						<?php _e( 'Create a Management API token using these steps', 'wp-auth0' ); ?>
-				  </a>
-					<?php _e( ' and paste it below:', 'wp-auth0' ); ?>
-				</p>
-
-				<input type="text" name="apitoken" class="js-a0-setup-input" autocomplete="off" required>
-				  <p>
-					  <small>
+		<form action="options.php" method="POST">
```

Comment:
Lots of whitespace and HTML adjustment. Documentation wording only saw minor changes.